### PR TITLE
PS-2456: support jobs batch endpoint

### PR DESCRIPTION
**Pull request by MiroCillik**
Depends on https://github.com/keboola/job-queue-internal-api/pull/82

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -28,7 +28,7 @@ services:
       - ./:/code
 
   internal-api:
-    image: keboolapes.azurecr.io/job-queue-internal-api:latest
+    image: keboolapes.azurecr.io/job-queue-internal-api:12463
```
Comment (joker806) [inline docker-compose.yml:31]: Nezapomen to potom zmenit zpatky na `latest` :)

File: src/Client.php [hunk 2]
```diff
@@ -93,6 +93,21 @@ public function createJob(JobInterface $job): JobInterface
...
+            $jobsData = json_encode($jobs, JSON_THROW_ON_ERROR);
+            $request = new Request('POST', 'jobs/batch', [], $jobsData);
+        } catch (JsonException $e) {
+            throw new ClientException('Invalid json of jobs: ' . $e->getMessage(), $e->getCode(), $e);
+        }
+        $result = $this->sendRequest($request);
+
+        return array_map(function ($jobData) {
+            return $this->jobFactory->loadFromExistingJobData($jobData);
+        }, $result);
```
Comment (joker806) [inline src/Client.php:108]:
```suggestion
        return array_map(fn(array $jobData) => $this->jobFactory->loadFromExistingJobData($jobData), $result);
```

File: tests/ClientFunctionalTest.php [hunk 3]
```diff
@@ -94,6 +94,88 @@ public function testCreateJob(string $kmsKeyId, string $keyVaultUrl, string $cip
...
+        self::assertNotEmpty($responseJobs);
+
+        /* @var Job $responseJob */
+        foreach ($responseJobs as $responseJob) {
+            $responseJobJson = $responseJob->jsonSerialize();
+            self::assertNotEmpty($responseJobJson['id']);
+            unset($responseJobJson['id']);
+            self::assertNotEmpty($responseJobJson['createdTime']);
+            unset($responseJobJson['createdTime']);
+            $storageClient = new StorageClient(
```
Comment (joker806) [inline tests/ClientFunctionalTest.php:138]: Tohle by asi melo jit mimo `foreach`

**Review by joker806 — APPROVED** [review]
Jen nejaky pitominky, jinak to vypada OK

File: tests/ClientFunctionalTest.php (hunk 3 above)
Comment (odinuv) [reply tests/ClientFunctionalTest.php]: done

File: docker-compose.yml (hunk 1 above)
Comment (odinuv) [reply docker-compose.yml]: done

**Pull request merged**